lib/lookup.json

```diff
@@ -99,7 +101,7 @@
   },
   "coffeescript": {
     "maintainers": ["jashkenas", "GeoffreyBooth"],
-    "skip": "rhel"
+    "skip": ["rhel", "aix"]
```

Comment:
I wasn’t aware of this . . . there’s not really any reason I can think of that CoffeeScript should fail in any environment.

Reply:
AIX: https://ci.nodejs.org/job/citgm-smoker/nodes=aix72-ppc64/2790/testReport/junit/(root)/citgm/coffeescript_v2_6_1/
```console
 npm ERR! code 1
 npm ERR! path /home/iojs/tmp/citgm_tmp/28cfdfc7-146e-4343-9f50-352435316c6c/coffeescript/node_modules/puppeteer
 npm ERR! command failed
 npm ERR! command sh -c node install.js
 npm ERR! /home/iojs/tmp/citgm_tmp/28cfdfc7-146e-4343-9f50-352435316c6c/coffeescript/node_modules/puppeteer/lib/cjs/puppeteer/common/assert.js:26
 npm ERR!         throw new Error(message);
 npm ERR!               ^
 npm ERR! 
 npm ERR! Error: Unsupported platform: aix
 npm ERR!     at Object.assert (/home/iojs/tmp/citgm_tmp/28cfdfc7-146e-4343-9f50-352435316c6c/coffeescript/node_modules/puppeteer/lib/cjs/puppeteer/common/assert.js:26:15)
 npm ERR!     at BrowserFetcher.setPlatform (/home/iojs/tmp/citgm_tmp/28cfdfc7-146e-4343-9f50-352435316c6c/coffeescript/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserFetcher.js:180:25)
 npm ERR!     at new BrowserFetcher (/home/iojs/tmp/citgm_tmp/28cfdfc7-146e-4343-9f50-352435316c6c/coffeescript/node_modules/puppeteer/lib/cjs/puppeteer/node/BrowserFetcher.js:164:14)
 npm ERR!     at PuppeteerNode.createBrowserFetcher (/home/iojs/tmp/citgm_tmp/28cfdfc7-146e-4343-9f50-352435316c6c/coffeescript/node_modules/puppeteer/lib/cjs/puppeteer/node/Puppeteer.js:185:16)
 npm ERR!     at downloadBrowser (/home/iojs/tmp/citgm_tmp/28cfdfc7-146e-4343-9f50-352435316c6c/coffeescript/node_modules/puppeteer/lib/cjs/puppeteer/node/install.js:42:46)
 npm ERR!     at download (/home/iojs/tmp/citgm_tmp/28cfdfc7-146e-4343-9f50-352435316c6c/coffeescript/node_modules/puppeteer/install.js:86:3)
 npm ERR! 
 npm ERR! Node.js v18.0.0-pre
 npm ERR! A complete log of this run can be found in:
 npm ERR!     /home/iojs/tmp/citgm_tmp/28cfdfc7-146e-4343-9f50-352435316c6c/home/.npm/_logs/2021-11-06T19_04_46_623Z-debug.log
```

Reply:
Ah, the _tests,_ that makes sense. If I wanted to get the tests running in AIX, say by disabling Puppeteer in AIX, is there some way I could launch an AIX environment to test that my changes work?